code/Task_List/app/src/main/java/aleksandr_logunov/task_list/helpers/TabSelectedListener.kt

```diff
@@ -3,6 +3,6 @@ package aleksandr_logunov.task_list.helpers
 import com.google.android.material.tabs.TabLayout
 
 interface TabSelectedListener : TabLayout.OnTabSelectedListener {
-    override fun onTabUnselected(tab: TabLayout.Tab?) {}
+    override fun onTabUnselected(tab: TabLayout.Tab?) {} // no need in interfaces {}
```

Comment:
It's a helper interface to make these two empty interfaces optional:

```
       tabs.addOnTabSelectedListener(object : TabSelectedListener {
                        override fun onTabSelected(tab: TabLayout.Tab?) {
                           // ...some code
                        }
                    })
```

instead of

```
        tabs.addOnTabSelectedListener(object : TabSelectedListener {
                        override fun onTabSelected(tab: TabLayout.Tab?) {
                             // ...some code
                        }

                        override fun onTabUnselected(tab: TabLayout.Tab?) {
                             // ...some code
                        }

                        override fun onTabReselected(tab: TabLayout.Tab?) {
                             // ...some code
                        }
                    })
```

I'll leave a comment about it.